data/transition-sites/directgov_los.yml

```
@@ -0,0 +1,6 @@
+---
+site: directgov_los
+whitehall_slug: government-digital-service
```

I think this should be DWP. The previous site was branded as a service by "jobcentreplus Department for Work and Pensions" – https://web.archive.org/web/20180920044754/http://los.direct.gov.uk

```suggestion
whitehall_slug: department-for-work-pensions
```

---

OK. I guess they might want to look at the analytics.